backend/src/Edna/Web/API.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE DataKinds #-}
+
+{-|
+Servant type-level specification for Edna API.
+-}
+module Edna.Web.API
+     ( EdnaAPI
+     , EdnaEndpoints (..)
+     , ednaAPI
+     ) where
```

Comment:
Formatting of exports is inconsistent and doesn't match style guide in most modules :cry:

Reply:
It was test to check your attention (to check that you are alive at all, I think ability to being able of such things determine your state being alive).

I will reformat to 2 spaces everywhere in the code.